lua/nvim-navic/init.lua

```diff
@@ -210,7 +217,7 @@ local lsp_num_to_str = {
 	[26] = "TypeParameter",
 }
 
-local config = {
+M.config = {
```

Comment:
Why does this internal table need to be exposed?

Reply:
reverted